src/Zope2/utilities/tests/test_zconsole.py

```diff
@@ -21,12 +21,19 @@
 
 test_script = """
 import sys
+import Testing
 
 
+
+def print_info():
+    # This tests the availability of global variables and imports.
+    print(sys.argv[1:])
+    print(Testing.__doc__[:7])  # `Set up`
```

Comment:
what about changing this into something less prone to change, like e.g. `math.pi`

Reply:
Maybe there is even something inside Zope which is more stable, e. g. the class name of `OFS.PropertyManager.PropertyManager`.

Reply:
I actually wanted to import something more complex with namespace etc. instead of standard library as the import of `sys` accidentally worked before, because it was in the scope of the exec call.